### PR TITLE
Show "Move All -> New Album" at all times. Actually move the photos

### PR DESCRIPTION
**Pull request by d7415**
when a new album is added this way.

Fixes LycheeOrg/Lychee#38

Rewrite of LycheeOrg/Lychee#84

### CODE REVIEW

File: scripts/album.js [hunk 1]
```diff
@@ -190,6 +190,50 @@ album.add = function() {
 
 };
 
+album.addandmove = function (photoIDs) {
+
+	const action = function(data) {
+
+		let title = data.title;
+
+		const isNumber = (n) => (!isNaN(parseFloat(n)) && isFinite(n));
```
Comment (ildyria) [inline scripts/album.js]: I think it should be parseInt() instead of parseFloat.

Comment (d7415) [reply scripts/album.js]: Possibly. That's taken directly from album.add (Line 156). Would you like me to change both?

Comment (ildyria) [reply scripts/album.js]: please do with `parseInt(x, 10)` (specify the base used in order to make sure that the use over different platform is the same). :)

Comment (d7415) [reply scripts/album.js]: Done :)

**Comment by ildyria** [comment]
Thank you. :)

**Pull request merged**